lib/puppet/provider/cisco_stp_global/nxapi.rb

```diff
@@ -0,0 +1,451 @@
+# The NXAPI provider for cisco_stp_global
```

Comment:
Please change this file to `cisco.rb` and `Puppet::Type.type(:cisco_stp_global).provide(:cisco)` - see #216.